mmedit/core/evaluation/metrics.py

```diff
@@ -192,6 +192,10 @@ def psnr(img1, img2, crop_border=0, input_order='HWC'):
     img1 = reorder_image(img1, input_order=input_order)
     img2 = reorder_image(img2, input_order=input_order)
 
+    if convert_to == 'y':
```

Comment:
In the signature of this function, `rgb` is set as the default args but missing from this `if-else` block. I recommand that you can define more behaviors here.

Reply:
In addition, the unit test should be added for this new functionality.